pkg/ingestor/parser/cyclonedx/parser_cyclonedx.go

```diff
@@ -475,13 +482,6 @@ func (c *cyclonedxParser) getAffectedPackages(ctx context.Context, vulnInput *mo
 	return &viList, nil
 }
 
-func (c *cyclonedxParser) getPackageElement(elementID string) []*model.PkgInputSpec {
```

Comment:
I made a few changes like this here and in the SPDX parser. I think those getter funcs are unneeded. I tested a bit here https://go.dev/play/p/QT6yz47_835